documentation/src/main/asciidoc/user_guide/chapter-76-Simple_Cache.adoc

```diff
@@ -0,0 +1,47 @@
+== Simple Cache
+
+Traditional local caches use the same architecture as clustered caches - interceptor stack. That way a lot of the functionality can be reused. However, if the advanced features are not needed and the performance is more important, the interceptor stack can be stripped away and simple cache can be used.
```

Comment:
Yeah, it's usually one paragraph per line, but that' because of the way it got converted to asciidoc. Putting each sentence on a separate line makes it much easier to review changes in a PR.